### PR TITLE
Custom fan curve rework documentation

### PR DESCRIPTION
**Pull request by Pokisiekk**
Signed-off-by: Kamil Pokornicki <kamil.pokornicki@3mdeb.com>

### CODE REVIEW

File: docs/unified-test-documentation/dasharo-compatibility/S30-fan-speed.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+1. Proceed with the
+   [Generic test setup: OS installation](../../generic-test-setup/#os-installation).
+1. Proceed with the
+   [Generic test setup: OS boot from disk](../../generic-test-setup/#os-boot-from-disk).
+
+## FAN001.001 CPU fan
+
+**Test description**
+
+This test aims to verify that the CPU fan RPM value is available to read.
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/S30-fan-speed.md:20]: We should change the description of the test case. We don't check in this case is the fan speed is available to read - we check if the fan works.

File: docs/unified-test-documentation/dasharo-compatibility/S30-fan-speed.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+1. Boot into the system.
+1. Log into the system by using the proper login and password.
+1. In the terminal window run the following command to get the RPM value of the
+   CPU fan:
+
+    ```bash
+    sensors | grep "CPU fan"
+    ```
+
+    > If the command fails, make sure the `system76_acpi` driver is available on
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/S30-fan-speed.md:44]: We should add this info to `Test setup` section.

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/unified-test-documentation/dasharo-compatibility/S30-fan-speed.md (hunk 1 above)
Comment (Pokisiekk) [reply docs/unified-test-documentation/dasharo-compatibility/S30-fan-speed.md]: Updated

File: docs/unified-test-documentation/dasharo-compatibility/S30-fan-speed.md (hunk 2 above)
Comment (Pokisiekk) [reply docs/unified-test-documentation/dasharo-compatibility/S30-fan-speed.md]: Replaced

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Pull request merged**